proto/evm/get_public_keys.proto

```diff
@@ -29,18 +38,22 @@ message GetPublicKeysIntiateRequest {
 
```

Comment:
in the comments of `derivation_paths` add the other documented derivation paths too.